Userland/Libraries/LibCompress/Zlib.h

```diff
@@ -76,7 +76,7 @@ class ZlibCompressor : public Core::Stream::Stream {
     static ErrorOr<ByteBuffer> compress_all(ReadonlyBytes bytes, ZlibCompressionLevel = ZlibCompressionLevel::Default);
 
 private:
-    ZlibCompressor(Core::Stream::Handle<Core::Stream::Stream>, ZlibCompressionLevel);
+    ZlibCompressor(Core::Stream::Handle<Core::Stream::Stream>, NonnullOwnPtr<Core::Stream::Stream>);
```

Comment:
Can we add some parameter names here? passing two streams, one that's a handle and one that's a NNOP is not as obvious as a stream and a compression level.